### ISSUE TITLE
Need some help with some usage way

### ISSUE THREAD

**Issue by pixeldin** [poucotm/Log-Highlight#51]
Thanks to the author, this is an excellent plugin, but I have encountered several problems, hope someone can help me. 
## ❓1. Plugin trigger way
The plugin will only work when the theme is set to the highlight option. When Triggering the log highlight in the right-click menu of the log file does not respond. The console log shown as below, Is there a problem with my usage?
![pplog](https://github.com/poucotm/Log-Highlight/assets/16974018/1ebfde58-7a35-4d20-a22a-bf518227d557)
**Actually, it only works when I use the global color schema setting like this:**
![pp-log-trigger](https://github.com/poucotm/Log-Highlight/assets/16974018/295c2464-3176-4cc3-909a-be8c19104410)

✔ It works so far this way.
![work](https://github.com/poucotm/Log-Highlight/assets/16974018/bbe0698e-076c-4842-9d14-fa6b2fae39dd)

## 🙈 2. The rule to write matching rules
Another problem is my current log file rules as:
```
[D] means debug, [I] means info level, [W] means warning, [E] means error
2023-05-19 21:48:46.993 [E] [L:327-main()] ====== #1 Cut for DEMO-base12-2 from 00:00:01 to 00:00:42, idx: #1, duration: 0min41s. 
2023-05-19 21:48:47.136 [W] [L:217-grep_for_key_and_before()] Grep pts, from 1(s) to 3(s) 
2023-05-19 21:48:49.148 [W] [L:240-grep_for_key_and_before()] From 1(s) to 3(s), got K_FRAME: 2.035000, BEF_KEY_FRAME: 2.002267 
2023-05-19 21:48:49.385 [E] [L:254-grep_for_key_and_before()] Grep 1(s) nearby result: 2.035000(K_FRAME) / 2.002267(BEF_KEY_FRAME) 
```
When I refer to the `Log Highlight.sublime-settings` file, what confuses me is how to modify the regularity of the configuration file for my log example?
I want `[E]` to match the `error` case and `[W]` to match the `warning` case.
In other words, how to modify the following configuration
```
...
"error" : { // specify name as yours
"enable"  : true,
"pattern" : [ // [ "begin regex", "end regex" ] or [ "match regex", "" ]
	// examples
	[ "^Error-\\[", "^\\s*[\\n]" ],                      // Error-[ ~ next empty line (multi-line)
	// [ "^{{{LINK}}}?[^\\r\\n]*?(?i)error", "[\\r\\n]" ],  // a line including case-insensitive 'error' with or without a link in front of 'error'
	["^{{{LINK}}}?[^\\r\\n]*?([Ee][Rr][Rr]?|\\[e\\])", "[\\r\\n]"],
	[ "^\\w+:\\s*\\*E", "[\\r\\n]" ]                     // ...: *E ... (single line)
	// <-- remove, change, add more patterns here -->
],
...
```
If you have any suggestions, please tell me :)🧡

**Comment by poucotm** [comment]
I've updated the package v2.2.3. Try again.
In order to match `[E]` to `error` case, try the following pattern :
```
"pattern" :
	[ "\\[E\\]", "[\\r\\n]" ]
],
```
You have to run `Generate Syntax & Theme` command after setting.

**Comment by pixeldin** [comment]
> I've updated the package v2.2.3. Try again. In order to match `[E]` to `error` case, try the following pattern :
> 
> ```
> "pattern" :
> 	[ "\\[E\\]", "[\\r\\n]" ]
> ],
> ```
> 
> You have to run `Generate Syntax & Theme` command after setting.

Great! Thanks, It works normal for log file now. And I want to match the `[I]` to `info` and `[D]` to `debug`, here is my config, I also `Generate Syntax & Theme` and restarted.
```json
// ...
"info" : {
	"enable"  : true,
	"pattern" : [
		[ "^Info-\\[", "^\\s*[\\n]" ],
		[ "\\[I\\]", "[\\r\\n]"],
		[ "^{{{LINK}}}?[^\\r\\n]*?(?i)info", "[\\r\\n]" ],
		[ "^\\w+:\\s*\\*I", "[\\r\\n]" ],
	],
	"color" : {
	        "base"  : ["#70991f", ""],
	        "link"  : ["#b36915", ""],
	        "quote" : ["#428a99", ""],
	},	
},
"debug" : {
	"enable"  : true,
	"pattern" : [
		[ "^Debug-\\[", "^\\s*[\\n]" ],
		[ "\\[D\\]", "[\\r\\n]"],
		[ "^{{{LINK}}}?[^\\r\\n]*?(?i)debug", "[\\r\\n]" ],
		[ "^\\w+:\\s*\\*D", "[\\r\\n]" ],
	],
	"color" : { 
		"base"  : ["#A1B347", ""],
		"link"  : ["#FD971F", ""],
		"quote" : ["#4F99D3", ""], 
	},
},
// ...
```
And it seems not match for `Info`/`[I]` or `debug`/`[D]`, so what's the problem?

![pattern_not_work](https://github.com/poucotm/Log-Highlight/assets/16974018/84afb6b8-9465-4e4e-b8b3-4977e4037a78)
